src/sentry/models/dashboard_permissions.py

```diff
@@ -0,0 +1,27 @@
+from __future__ import annotations
+
+from django.db import models
+
+from sentry.backup.scopes import RelocationScope
+from sentry.db.models import Model, region_silo_model
+from sentry.db.models.base import sane_repr
+
+
+@region_silo_model
+class DashboardPermissions(Model):
+    """
+    Edit permissions for a Dashboard.
+    """
+
+    __relocation_scope__ = RelocationScope.Organization
+
+    is_creator_only_editable = models.BooleanField(default=False)
+    dashboard = models.OneToOneField(
+        "sentry.Dashboard", null=True, blank=True, on_delete=models.CASCADE
+    )
```

Comment:
In this case, I don't think we to define `null=True` and `blank=True`, we'll always want the dashboard field to be filled out when creating DashboardPermissions. It was only necessary on the `Dashboard` side because a dashboard might not necessarily have perms